compute_results.py

```diff
@@ -0,0 +1,219 @@
+                duration_bruxism = int(window_length_bruxism * raw.info['sfreq'])  # in sample
+                window_length_bruxism = duration_bruxism / raw.info['sfreq']  # recompute exact window_length
+                duration_MEMA = duration_factor_MEMA * duration_bruxism
+                window_length_MEMA = duration_MEMA / raw.info['sfreq']  # recompute exact window_length
+                duration_OMA = duration_factor_OMA * duration_bruxism
+                duration_Impedance = duration_factor_Impedance * duration_bruxism
+                n_adaptive_bruxism = int(n_adaptive_bruxism / window_length_bruxism)
+
+                log = {}
+
+                picks_chan_bruxism = []
+                for elm in ind_picks_chan:
+                    picks_chan_bruxism.append(raw.info["ch_names"][elm])
+
+                picks_imp = []
+                for elm in ind_picks_imp:
+                    picks_imp.append(raw.info["ch_names"][elm])
+
+                DO_BRUXISM = len(picks_chan_bruxism) > 0 and bruxism
+                DO_MEMA = file in mema_files and mema
+
+                # ----------------- Preprocessing ----------------------------------------------------
+                print(f"preprocess...", end=" ")
+                valid_labels_bruxism = []
+                valid_labels_MEMA = []
+
+                print(f"Bruxism(", end="")
+                tmp = time()
+                if DO_BRUXISM:
+                    is_good_kwargs = dict(ch_names=picks_chan_bruxism,
+                                          rejection_thresholds=dict(emg=5e-05),  # two order of magnitude higher q0.01
+                                          flat_thresholds=dict(emg=1e-09),  # one order of magnitude lower median
+                                          channel_type_idx=dict(emg=[i for i in range(len(picks_chan_bruxism))])
+                                          )
+                    epochs_bruxism, valid_labels, log["bruxism"] = preprocess(raw, duration_bruxism,
+                                                                                      duration_bruxism,
+                                                                                      picks_chan=picks_chan_bruxism,
+                                                                                      is_good_kwargs=is_good_kwargs)
+                    valid_labels_bruxism.append(valid_labels)
+                    print(f"done)", end=" ")
+                else:
+                    print(f"skipped)", end=" ")
+
+                print(f"MEMA(", end="")
+                if DO_MEMA:
+                    epochs_MEMA, valid_labels, log["MEMA"] = preprocess(raw, duration_MEMA, duration_MEMA,
+                                                                             picks_chan=['Airflow'])
+                    valid_labels_MEMA.append(valid_labels)
+                    print(f"done)", end=" ")
```

Comment:
What is the purpose of these lines? there shouldn't be a is_good processing on MEMA no?

Reply:
preprocess is just used here to pick the channel and do the epoching
valid_labels should be all True